src/Runner.Worker/ActionCommandManager.cs

```diff
@@ -1,4 +1,4 @@
-﻿using GitHub.DistributedTask.Pipelines.ContextData;
```

Comment:
some of these files had a UTF-8 byte-order-mark (BOM).  The linter complained about the BOM, so I changed the file encoding to be BOMless.